index.js

```diff
@@ -5,11 +5,13 @@
 var util = require('util');
 var winston = require('winston');
 var winex = require('./winex');
+const { format } = require('logform');
```

Comment:
Can you leave a comment that this isn't listed as a dep in package.json, but it is a dep of winston, and we want to use whatever version winston is using?